content/docs/quickstart/go.md

```diff
@@ -3,122 +3,54 @@ title: Go Quick Start
 layout: quickstart
 short: Go
 description: This guide gets you started with gRPC in Go with a simple working example.
-protoc-version: 3.11.4
 ---
 
-<style type="text/css">
-    ol ol { list-style-type: lower-alpha !important; }
-</style>
-
 ### Prerequisites
 
-- **[Go][]**, any one of the **three latest major** [releases of Go][]. For
-  installation instructions, see Go's [Getting Started][] guide.
-
-#### gRPC
-
-Use the following command to install gRPC as a [module][]:
-
-```sh
-$ export GO111MODULE=on # Enable module-aware mode
-$ go get google.golang.org/grpc@{{< param grpc_release_tag >}}
-```
-
-#### Protocol Buffers
-
-While not mandatory, gRPC applications usually leverage [Protocol Buffers][pb]
-for service definitions and data serialization, and the example code uses
-[proto3][].
+- **[Go][]**, any one of the **three latest major** [releases of Go][].
 
- 1. Install the `protoc` compiler:
+  For installation instructions, see Go's [Getting Started][] guide.
 
-     1. Download a zip file of the latest version of pre-compiled binaries for
-        your operating system from [github.com/google/protobuf/releases][]
-        (`protoc-<version>-<os><arch>.zip`). For example:
+- **[Protocol buffer][pb] compiler**, `protoc`, [version 3][proto3].
 
-        ```sh
-        $ PB_REL="https://github.com/protocolbuffers/protobuf/releases"
-        $ curl -LO $PB_REL/download/v{{< param protoc-version >}}/protoc-{{< param protoc-version >}}-linux-x86_64.zip
-        ```
+  For installation instructions, see [Protocol Buffer Compiler
+  Installation][pbc-install].
 
-     2. Unzip the file under `$HOME/.local` or a directory of your choice. For
-        example:
+- **Go plugin** for the protocol compiler:
 
-        ```sh
-        $ unzip protoc-{{< param protoc-version >}}-linux-x86_64.zip -d $HOME/.local
-        ```
+   1. Install the protocol compiler plugin for Go (`protoc-gen-go`) using the
+      following command:
 
-     3. Update your environment's path variable to include the path to the
-        `protoc` executable. For example:
+      ```sh
+      $ go get github.com/golang/protobuf/protoc-gen-go@v1.3.5
```

Comment:
Is it a good idea to specify a patch release here?  If there are bug fixes, a later version of the same minor release would be preferred.  But maybe we shouldn't specify any version?  Future minor versions should always be backward compatible.

... And, they've now deprecated this package, so we'll have to update these instructions to use the v2 implementation once we're ready -- I think we're blocked on https://github.com/grpc/grpc-go/pull/3453 (me) for this.

Reply:
> ... so we'll have to update these instructions to use the v2 implementation once we're ready ...

Right. I see this PR as another iterative improvement of the go quick start. Once we're in a position to use the new plugin, I'll update the instructions and drop the `@v1.3.5` version specification.

Reply:
How about drop the `.5` for now, anyway, so we don't pin to a specific version?  I doubt they'll ever release 1.3.6 now that they're on 1.4.x, but if they do it will be for a very good reason.

Reply:
Done.